Libraries/Components/Button.js

```diff
@@ -154,6 +154,7 @@ type ButtonProps = $ReadOnly<{|
    * For arrow keys, add "ArrowLeft", "ArrowRight", "ArrowUp", "ArrowDown",
    */
   validKeysUp?: ?Array<string>,
+
```

Comment:
This is left over from adding and removing the prop... but should probably be here so I'll keep it